src/modm/platform/i2c/bitbang/bitbang_i2c_master.hpp.in

```diff
@@ -51,9 +51,9 @@ class BitBangI2cMaster : public modm::I2cMaster
 
 public:
 	// start documentation inherited
-	template< class... Signals, ResetDevices reset = ResetDevices::Standard >
+	template< class... Signals >
 	static void
-	connect(PullUps pullups = PullUps::External);
+	connect(PullUps pullups = PullUps::External, ResetDevices reset = ResetDevices::Standard);
```

Comment:
The interface also needs to be adapted:
https://github.com/modm-io/modm/blob/develop/src/modm/architecture/interface/i2c_master.hpp#L89-L91

Reply:
:+1:, I'll fix it.